pcdscalc/be_lens_calcs.py

```diff
@@ -201,17 +193,17 @@ def get_lens_set(set_number_top_to_bot, filename=None):
     if os.stat(filename).st_size == 0:
         logger.error('The file is empyt: %s', filename)
         return
-    with open(filename, 'rb') as lens_file:
-        sets = np.load(lens_file, allow_pickle=True)
-        if set_number_top_to_bot not in range(1, sets.shape[0]):
+    with open(filename) as lens_file:
+        sets = json.loads(lens_file.read())
+        if set_number_top_to_bot not in range(1, len(sets)):
```

Comment:
This and other errors in the function should probably be bumped up to exceptions (`ValueError`) as they'd break the calling function by returning `None`